Libplanet.Tests/AddressTest.cs

```diff
@@ -311,6 +312,24 @@ public void Serializable()
             Assert.Equal(expected, deserialized);
         }
 
+        [Fact]
+        public void TypeConverter()
+        {
+            TypeConverter converter = TypeDescriptor.GetConverter(typeof(Address));
+            var address = new Address("0123456789ABcdefABcdEfABcdEFabcDEFabCDEF");
+            Assert.True(converter.CanConvertFrom(typeof(string)));
+            Assert.Equal(
+                address,
+                converter.ConvertFrom("0x0123456789ABcdefABcdEfABcdEFabcDEFabCDEF"));
+            Assert.Equal(
+                address,
+                converter.ConvertFrom("0123456789ABcdefABcdEfABcdEFabcDEFabCDEF"));
+            Assert.True(converter.CanConvertTo(typeof(string)));
+            Assert.Equal(
+                "0123456789ABcdefABcdEfABcdEFabcDEFabCDEF",
+                converter.ConvertTo(address, typeof(string)));
```

Comment:
I'm not sure about how type conversion works here so this is just a question, but is it okay to not implement some kind of validation for the string to be converted to an Address instance, or a fail test case? (PublicKeyTest too)

Reply:
They will throw exceptions such as `ArgumentOutOfRangeException` or `FormatException`, and Cocona treat it as a wrong option value if whatever they throw.

Reply:
On second thoughts, however, it would be better to have tests for exceptional cases as well.  I'm going to add them.

Reply:
@tkiapril Amended.  Please take a look again!